### PR TITLE
Benchmark for `CondaRepodata.Remove`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #5 
Set up benchmarks project and added benchmark for `CondaRepodata.Remove`.

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Conda adapter benchmarks
+
+To run benchmarks:
+ 1. Install snapshot locally of `conda-adapter`: `mvn install`
+ 2. Build `conda-bench` project: `mvn package -f ./benchmarks`
+ 4. Create directory for tests and copy test resources to this directory
```
Comment (genryxy) [inline benchmarks/README.md:6]: @olenagerasimova why the third section is missed? Is it necessary also to specify path to dependencies?

File: benchmarks/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ 5. Run benchmarks with `env BENCH_DIR=${test-dir} java -cp "benchmarks/target/benchmarks.jar" org.openjdk.jmh.Main ${bench-name}`, where `${test-dir}` is a directory with test data, and `${bench-name}` is a benchbmark name.
+
+## Benchmarks
```
Comment (genryxy) [inline benchmarks/README.md:9]: @olenagerasimova probably also add information about added test

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see comments

File: benchmarks/README.md (hunk 2 above)
Comment (olenagerasimova) [reply benchmarks/README.md:10]: @genryxy I'll add in the next PR along with github action, need to make sure link to download test resource is working

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected and answered

File: benchmarks/README.md (hunk 1 above)
Comment (genryxy) [reply benchmarks/README.md:7]: @olenagerasimova does it work when dependencies are not specified in `java -cp "..."`

Comment (olenagerasimova) [reply benchmarks/README.md:7]: @genryxy yes, it works for me

Comment (genryxy) [reply benchmarks/README.md:7]: @olenagerasimova okay, got it. Sorry for misleading

**Review by genryxy — APPROVED** [review]

**Pull request merged**